Worker.Dockerfile

```diff
@@ -17,10 +17,15 @@ RUN apt-get update && \
 RUN python3 -m venv /venv
 
 # Copy the requirements file to the container
-COPY requirements.txt /venv/
+COPY requirements.txt /tmp/requirements.txt
```

Comment:
Sorry, one more nit / question: The original code did not name the file to copy to, but just the target directory. Should we do the same here and just say `COPY requirements.txt /tmp/`?